stories/dooboo-ui/Card.stories.tsx

```diff
@@ -89,7 +89,7 @@ function Default(): React.ReactElement {
         }}
         title={'This is title'}
         titleStyle={{ display: 'flex', flexDirection: 'column' }}>
-        <Text>I don't have Image</Text>
+        <Text>{"I don't have Image"}</Text>
```

Comment:
Just question : Why use '{ }' ?

Reply:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md
to remove this eslint error

Reply:
Ok.!!